spec/requests/api/provision_requests_spec.rb

```diff
@@ -105,6 +105,140 @@ def app
     end
   end
 
+  context "AWS advanced provision requests" do
+    let(:aws_dialog)     { FactoryGirl.create(:miq_dialog_aws_provision) }
+    let(:ems)            { FactoryGirl.create(:ems_amazon_with_authentication) }
+    let(:template) do
+      FactoryGirl.create(:template_amazon, :name => "template1", :ext_management_system => ems)
+    end
+    let(:flavor) do
+      FactoryGirl.create(:flavor_amazon, :ems_id => ems.id, :name => 't2.small', :cloud_subnet_required => true)
+    end
+    let(:az)             { FactoryGirl.create(:availability_zone_amazon) }
+    let(:cloud_network1) { FactoryGirl.create(:cloud_network, :ems_id => ems.id, :enabled => true) }
+    let(:cloud_subnet1)  { FactoryGirl.create(:cloud_subnet, :cloud_network_id => cloud_network1.id) }
+
+    let(:provreq_body) do
+      {
+        "template_fields" => {"guid" => template.guid},
+        "requester"       => {"user_name" => api_config(:user)}
```

Comment:
The requester tab has 3 required fields by default: email, owner_first_name and owner_last_name.

```ruby
"requester"       => {
  "owner_first_name" => api_config(:user_name),
  "owner_last_name"  => api_config(:user_name),
  "owner_email"      => "user@example.test"
}
```